templates/taxi/car_detail.html

```diff
@@ -1,12 +1,17 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <h1>Car: {{ car.model }}</h1> 
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+    
   <h1>Drivers</h1>
   <hr>
   <ul>
   {% for driver in car.drivers.all %}
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
   {% endfor %}
   </ul>
+    <a href="{% url 'taxi:car-update' car.pk %}" class="btn btn-primary"></a>
```

Comment:
The update button is missing a label. Consider adding text like 'Update' inside the `<a>` tag to make it clear to users what the button does.